src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java

```diff
@@ -46,67 +46,109 @@
 public class EDDifferentialPrivacy extends ImplicitPrivacyCriterion {
     
     /** SVUID */
-    private static final long        serialVersionUID = 242579895476272606L;
+    private static final long              serialVersionUID = 242579895476272606L;
 
     /** Parameter */
-    private final double             epsilon;
+    private final double                   epsilonAnon;
     /** Parameter */
-    private final double             delta;
+    private final double                   epsilonSearch;
     /** Parameter */
-    private final int                k;
+    private final int                      steps;
     /** Parameter */
-    private final double             beta;
+    private final double                   delta;
     /** Parameter */
-    private DataSubset               subset;
+    private final int                      k;
     /** Parameter */
-    private transient DataManager    manager;
+    private final double                   beta;
     /** Parameter */
-    private transient boolean        deterministic    = false;
+    private DataSubset                     subset;
     /** Parameter */
-    private DataGeneralizationScheme generalization;

```

Comment:
Thanks for pointing this out, I've created a new class which implements data-dependent DP and performed tests to assure that proper deserialization of previously serialized EDDifferentialPrivacy instances works.